.github/workflows/clean-expired-jobs.yaml

```diff
@@ -24,14 +24,14 @@ jobs:
         python3 scripts/clean_jobs.py
 
     - name: Run URLs-checker
-      uses: urlstechie/urlchecker-action@0.2.1
+      uses: urlstechie/urlchecker-action@0.0.25
```

Comment:
So does this go back to a much older version?

Reply:
Nope, it goes forward - the versioning scheme changed in the last tag.

Reply:
:laughing: How helpful of them, thank you for clarifying.

Reply:
Yes this is an attempt to make syncing between urlchecker Python and action more logical - should have done it from the beginning! We will skip entirely the group/namespace that we were using.

Reply:
And @mtbc don't laugh too hard this is one of my libraries… :/

Reply:
My apologies, thank you for trying to make the versioning more useful!

Reply:
haha totally no worries! I'm aware that I might be making a package manager cry :laughing: but on the other hand, it's just a GitHub Action version so possibly slightly less offensive?